rudder-core/src/main/scala/com/normation/rudder/services/path/PathComputer.scala

```diff
@@ -41,21 +41,21 @@ import com.normation.rudder.services.policies.nodeconfig.NodeConfiguration
 
 /**
  * Utilitary tool to compute the path of a server promises (and others information) on the rootMachine
- *
- * @author nicolas
+
  *
  */
 trait PathComputer {
 
   /**
-   * Compute the base path for a server, i.e. the full path on the root server to the data
-   * the searched server will fetch, and the backup folder
-   * Finish by the server name, with no trailing /
-   * Ex : /opt/hive/cfserved/serverA/served/serverB, /opt/hive/backup/serverA/served/serverB
-   * @param searchedNode : the server we search
-   * @return
+   * Compute
+   *  the path of promises for a node, i.e. the full path on the root server to the data
+   *  the new folder, i.e. the full path where promises are written and checked before being updated
+   *  the backup folder, the path were promises are backuped
+   * Finish with no trailing /
+   * It does not contain the root machine, except if we search the root machine
+   * Ex : /var/rudder/share/uuid-a/share/uuid-b/rules,/var/rudder/share/uuid-a/share/uuid-b/rules.new, /var/rudder/backup/uuid-a/share/uuid-b
```

Comment:
Could you synchronize the comment witht the one on implementation ? Or only put the comment in the interface, if you prefer ?